pycharm/install.bash

```diff
@@ -1,4 +1,5 @@
 #! /usr/bin/env bash
+[[ "$CI" == "true" ]] && return 0
```

Comment:
Which idiot does install this in CI?

Reply:
We test all modified targets in CI. Some just don't make sense to test in CI.